packages/utils/src/loader/cdn-loader.ts

```diff
@@ -49,48 +49,33 @@ export class CDNLoader {
    * @param href The location of the SVG to load
    * @returns Promise of the SVG body
    */
-  private async loadContent (href: string): Promise<XMLHttpRequest> {
+  private async loadContent (href: string): Promise<Response> {
     try {
-      const response = await this.xmlRequest(href);
-      return response;
+      return await fetch(href);
     }
     catch (e) {
       // Failed response...
       this.responseCache.delete(href);
```

Comment:
Why need to delete response cache.

Reply:
To delete from the cache, the cache should keep only success request. Fail requests like network problem, offline, or other issues shouldn't store because the request isn't successful. Fail response may take error or doesn't exist.